extract-docker-tag/main.go

```diff
@@ -78,7 +81,7 @@ func extract(action *githubactions.Action) (*result, error) {
 		// always add tag prefix and name suffix to prevent clashes on "main", "latest", etc
 		branch := action.Getenv("GITHUB_HEAD_REF")
 		parts = strings.Split(strings.ToLower(branch), "/") // for branches like "dependabot/submodules/XXX"
-		result.tag = "pr-" + parts[len(parts)-1]
+		result.tags = append(result.tags, "pr-"+parts[len(parts)-1])
```

Comment:
I think simple assignment will be better there

Reply:
Made the first assignment simple through that function.